templates/hail_layout.ld

```diff
@@ -0,0 +1,17 @@
+/* Layout for the Hail board, used by the examples in this repository. */
```

Comment:
Should the layout files be symlinks to the top-level directory files (or the other way around), to make sure they are kept in sync? Same for `rust-toolchain`.